### ISSUE TITLE
marvin: URL does not exist 

### ISSUE THREAD

**Issue by sheepforce** [NixOS/nixpkgs#125236]
**Describe the bug**
The `marvin` package is broken at least in `nixos-21.05` and `nixpkgs-unstable` as the URL throws a 404 error.
```
trying http://dl.chemaxon.com/marvin/21.3.0/marvin_linux_21.3.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download marvin-21.3.0.deb from any mirror
builder for '/nix/store/aj4w5n1x2jqw10k41nfibl005s45b375-marvin-21.3.0.deb.drv' failed with exit code 1
```

**To Reproduce**
`nix-shell -I https://github.com/NixOS/nixpkgs/archive/84aa23742f6c72501f9cc209f29c438766f5352d.tar.gz -p marvin`

**Expected behavior**
Correct evaluation of `marvin` without 404 errors.

**Notify maintainers**
@fusion809 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.119, NixOS, 20.09.4169.ac60476ed94 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.11`
 - channels(root): `"nixos-21.05beta555.d25ea6a0d2a"`
 - channels(phillip): `"nixpkgs-21.05pre291442.278bcdce1f0"`
 - nixpkgs: `/home/phillip/.nix-defexpr/channels/nixpkgs`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: 
  - marvin
# a list of nixos modules affected by the problem
module:
```

**Comment by fusion809** [comment]
Yep, the URL doesn't exist for this old version of Marvin. The latest version of Marvin (21.9) downloads fine though. I suggest you use a different channel if you want to install Marvin.

**Comment by sheepforce** [comment]
Alright. It didn't make it to the nixpkgs-unstable channel yet, but I see it is in the repo. Nevertheless, it is broken in 21.05. Could it be backported or be marked broken?

**Comment by collares** [comment]
Fixed in #125389.
